core/embed/prodtest/README.md

```diff
@@ -240,6 +282,26 @@ FIRMWARE VERSION
 OK 0.2.6
 ```
 
+### BOOTLOADER_VERSION
```

Comment:
The command name is `BOOTLOADER VERSION` (without the undescore).

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/b7964bb62095997f462700d08ca24db7143c35af